frontend/src/pages/devices/List.tsx

```diff
@@ -78,23 +79,23 @@ export const List = () => {
                 <tbody>
                     {entities.map(entity =>
                         <tr key={ entity.id }>
-                            <th scope="row">
+                            <th className="align-middle">
                                 <Link to={"/devices/" + entity.id}>
                                     <Button variant="link">{ entity.rawDevice.name }</Button>
                                 </Link>
                             </th>
-                            <td>{ renderEntityStatus(entity) }</td>
-                            <td>{ renderEntityBatteryLevel(entity) }</td>
-                            <th scope="row">
+                            <td className="align-middle text-left">{renderEntityStatus(entity)}</td>
```

Comment:
Could you show what this looks like with and w/o the `text-left`?

Reply:
It doesn't appear to do anything, I think it was an attempt to bring the status column left more because it doesn't line up correctly. I spent a good chunk of time trying to make it better but I realized that the other columns were like this as well so I decided it was good enough.
![look](https://user-images.githubusercontent.com/54916525/196293266-ec21ce14-8a21-4aba-8fb1-e5298a85c830.png)